### ISSUE TITLE
Preload images for tomorrow in background

### ISSUE THREAD

**Issue by toyg** [gowong/tabtrekker#94]
I love this add-on, but it has one issue: it takes some time to load when you start the browser, presumably because you're downloading today's pictures.

Wouldn't it be possible to have the extension cache one or more days of pictures, silently in the background? This would hopefully speed up first load, and even give you a buffer for short offline periods.

**Comment by gowong** [comment]
Yes, this is definitely possible and would be a good enhancement! 👍
